Nil pointer dereference using

Using `goapi -gopath github.com/go-clang/v3.4` produces a nil pointer dereference:

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x4035a1]

goroutine 1 [running]:
panic(0x5e83e0, 0xc420010210)
        /home/zimmski/go/go/src/runtime/panic.go:500 +0x1a1
main.(*Walker).export(0xc420016540, 0x0)
        /home/zimmski/go/src/github.com/vbatts/goapi/goapi.go:236 +0x51
main.main()
        /home/zimmski/go/src/github.com/vbatts/goapi/goapi.go:176 +0x567
```

--- a/goapi.go
+++ b/goapi.go
@@ -233,6 +233,9 @@ func (w *Walker) export(pkg *types.Package) {
 	if *verbose {
 		log.Println(pkg)
 	}
+	if pkg == nil {
+		return
+	}
 	pop := w.pushScope("pkg " + pkg.Path())
 	w.current = pkg
 	scope := pkg.Scope()

